code/game/objects/items/tools/misc_tools.dm

```diff
@@ -275,6 +275,25 @@
 	desc = "It's an invisible pen marker."
 	pen_colour = "white"
 
+/obj/item/tool/pen/fountain
+	desc = "A luxurious fountain pen, embossed with gold accents. Its intricate mechanics allow the user to switch between various ink colors with a simple twist."
+	name = "fountain pen"
```

Comment:
Name above desc

Reply:
Do you want me to apply this to the rest of the file or should mine be the only one done differently?

![image](https://github.com/cmss13-devs/cmss13/assets/24533979/614652f2-6e83-4eab-9e18-8955b6bb9f13)